bindings/Fortran/CMakeLists.txt

```diff
@@ -49,15 +52,18 @@ target_include_directories(adios2_f
 
 target_link_libraries(adios2_f PRIVATE adios2)
 
+if(ADIOS2_HAVE_FORTRAN_SUBMODULES)
+  target_compile_definitions(adios2_f PRIVATE "$<$<COMPILE_LANGUAGE:Fortran>:ADIOS2_HAVE_FORTRAN_SUBMODULES>")
+endif()
 
 if(ADIOS2_HAVE_MPI)
+  target_compile_definitions(adios2_f PRIVATE "$<$<COMPILE_LANGUAGE:Fortran>:ADIOS2_HAVE_MPI_F>")
```

Comment:
Is there a scenario where these files would ever be built with `ADIOS2_HAVE_MPI_F` not defined?

Reply:
This is needed to tell the *other* sources that MPI parts will be present.

Reply:
These two non-MPI sources need to know whether the MPI sources will be included:

* `bindings/Fortran/modules/adios2_adios_init_mod.F90`
* `bindings/Fortran/modules/adios2_io_open_mod.F90`